lib/util/src/platform.c

```diff
@@ -266,10 +265,10 @@ hse_platform_fini(void)
     kmem_cache_fini();
     perfc_fini();
     vlb_fini();
-    logging_fini();
     hse_timer_fini();
     hse_cgroup_fini();
     dt_fini();
+    logging_fini();
```

Comment:
This just goes to show how over-complicated logging was before you eviscerated it!  :-)